sbc_translation/models/translate_connector.py

```diff
@@ -0,0 +1,159 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+import datetime
+
+from openerp.addons.mysql_connector.model.mysql_connector \
+    import mysql_connector
+
+
+class TranslateConnect(mysql_connector):
+    """ Contains all the utility methods needed to talk with the MySQL server
+        used by translate platform, as well as all mappings
+        from OpenERP fields to corresponding MySQL fields.
+    """
+
+    def __init__(self):
+        super(TranslateConnect, self).__init__(
+            'mysql_translate_host',
+            'mysql_translate_user',
+            'mysql_translate_pw',
+            'mysql_translate_db')
+
+        self.current_time = datetime.datetime.now()
+
+    def upsert_text(self, sponsorship, letter):

```

Comment:
You could only pass `letter` as argument, and sponsosrhip would be `letter.sponsorship_id`